zulip_bots/zulip_bots/finder.py

```diff
@@ -21,3 +24,14 @@ def import_module_from_source(path: Text, name: Optional[Text]=None) -> Any:
         loader.exec_module(module)
 
     return module
+
+def resolve_bot_path(name: Text) -> Tuple[Text, Text]:
+    if os.path.isfile(name) or os.path.isfile(name + '.py'):
+        bot_path = os.path.abspath(name) if os.path.isfile(name) else os.path.abspath(name + '.py')
+        bot_name = splitext(basename(bot_path))[0]
+    else:
+        bot_path = os.path.abspath(os.path.join(current_dir, 'bots', name, name + '.py'))
+        bot_name = name
+    sys.path.insert(0, os.path.dirname(bot_path))
```

Comment:
I'm not sure if we want this function to also insert the bot path into the environment PATH, it's not something I'd expect a function named `resolve_bot_path` to do. Maybe we can find a more suitable name for it. @showell do you have an opinion on this?